src/website/views/components/activity.pug

```diff
@@ -0,0 +1,1105 @@
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
+                  <li>
+                    <div>
+                      <div class="a-icon -text--success">
+                        <svg>
+                          <use xlink:href="#icon-check-alt"></use>
+                        </svg>
+                      </div>
+                      Instance successfully deployed
+                    </div>
+                    <div class="-time">1 minute ago</div>
+                  </li>
+                  <li>
+                    <div>
+                      <div class="a-icon -text--success">
+                        <svg>
+                          <use xlink:href="#icon-check-alt"></use>
+                        </svg>
+                      </div>
+                      Machine active-directory-vbo8o-1 successfully deployed
+                    </div>
+                    <div class="-time">2 minute ago</div>
+                  </li>
+                  <li>
+                    <div>
+                      <div class="a-icon -text--muted">
+                        <svg>
+                          <use xlink:href="#icon-circle-plus-outline"></use>
+                        </svg>
+                      </div>
+                      Executing install:Active Directory in machine active-directory-vbo8o-1
+                    </div>
+                    <div class="-time">2 minute ago</div>
+                  </li>
+                  <li>
+                    <div>
+                      <div class="a-icon -text--muted">
+                        <svg>
+                          <use xlink:href="#icon-circle-plus-outline"></use>
+                        </svg>
+                      </div>
+                      Updating machine IPs
+                    </div>
+                    <div class="-time">3 minute ago</div>
+                  </li>
+                </ul>
+              </div>
+              <div class="m-activity__end-footer">04/21/2019</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+h4 Feed with stories and operation icons
+.example.-mb--3
+  .-p--5
+    .m-activity.-feed.-stories
+      .m-activity__day
+        .m-activity__content
+          .m-activity__item
+            .m-activity__start
+              div
+                .a-icon
+                  svg
+                    use(xlink:href='#icon-circle-play-outline')
+                .a-avatar
+                  img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
+                |  deployed
+                strong  Active Directory
+              .m-activity__end-content
+                ul.-stories
```

Comment:
@davidojedacl is this needed for any purpose?  If don't, please remove it.

Reply:
done